x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -149,20 +148,21 @@ export class MBMapContainer extends React.Component {
     }
   }
 
-  assignSizeWatch() {
+  _initResizerChecker() {
+    let animationFrameTimeout;
     this._checker = new ResizeChecker(this.refs.mapContainer);
-    this._checker.on('resize', (() => {
-      let lastWidth = window.innerWidth;
-      let lastHeight = window.innerHeight;
-      return () => {
-        if (lastWidth === window.innerWidth
-          && lastHeight === window.innerHeight && this._mbMap) {
-          this._mbMap.resize();
+    this._checker.on('resize', () => {
```

Comment:
nice, this is more clean too.

I don't think we need to wrap this in an animation-frame callback. It assumes we'd get resize events at a rate exceeding 60fps, and that mapbox would not be optimizing for `map.resize()` method calls internally. It'd suprise me if both were true. I think we should be fine just calling resize on each event.

Reply:
@thomasneirynck I have removed the animation frame. Looks like EUI has a [ResizeObserver](https://elastic.github.io/eui/#/utilities/resizeobserver) that we could use instead of ui/public/resize_checker. The problem is that ResizeObserver was only added in EUI 7.2.0 and Kibana is currently on EUI 7.1.0. I will make a separate PR to clean this up more and replace resize_checker (which is based on angular) with ResizeObserver once Kibana EUI has been upgraded.

Reply:
Or, I can just revert the changes to this file and the resize issue can be completely addressed in a separate PR since resize_checker is going to be replaced anyways.

Reply:
it's good to fix it here imho

OK with new PR where we migrate to eui's ResizeObserver later

Reply:
There was some reason we needed this previously but I'm comfortable with getting rid of it and letting it re-emerge if necessary. Really might not be needed anymore.